specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/examples/Get_Certs.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "tenantBaseUrl": "'https://mytest.attest.azure.net'",
+    "api-version": "2018-09-01-preview",
+    "tee": "SgxEnclave"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "keys": [
+          {
+              "kid": "TCuR47Kzk4QCu-7hzSDIEGGKyy6nmedtaBYeOiUQf1o",
+              "kty": "RSA",
+              "n": "rw8F5WjDKzq6WGBCS77pTlZaspSc_YZ21Brn2TcGFQscfXXGlwi-U8s-BODWTyK_a1w5VPgl0i-Llw-B5PdrcT-QfUbPvsiyuOqxiYEFrEINvYRg-uL6eUfnMlMeG7qscd6SsUyYi-Yc-VoRSMFv1zU4ZFa0EzxyTmbV9UuDMHaeKhBuGGpxVtZWl5ZVI6sPG-_RAWcP3rCFyTx_jwE7VqSc8LEHW7zbIAdgMtKnyGRjN3DwFF1h7jHimHZfMHi2n2PB6hxmW7ICEB1B6n3NHpacP2YJ0dJEWyRbIEPvrgrEUFleGdqkBUpvCcSjGpY0hlTNWt2KTMP5R5wchRGMGw",
```

Comment:
The body of our get_certs response will never return a kty of "RSA" - we'll always return a kty of X5C.

Reply:
This is an example I got from postman return result

Reply:
When I go to: 
https://sqlclientattestvsm.us.attest.azure.net/certs
I get:
`{"keys":[{"x5c":["MIIDXzCCAkegAwIBAgIQGCXfIe4c6Y5DI0ubtcwppDANBgkqhkiG9w0BAQsFADA5MTcwNQYDVQQDEy5odHRwczovL3NxbGNsaWVudGF0dGVzdHZzbS51cy5hdHRlc3QuYXp1cmUubmV0MB4XDTE5MTIxMDAwMzIyMFoXDTIwMTIxMDAwMzIyMFowOTE3MDUGA1UEAxMuaHR0cHM6Ly9zcWxjbGllbnRhdHRlc3R2c20udXMuYXR0ZXN0LmF6dXJlLm5ldDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBANAXUJFngKcq2jczW5haSeprJGj0Nm0Qn8PA+fePmPuo6d8BMFKHD5dzkkQ4xYSmWMZV5WisaeDqnaJ9FeQcnGTVgZvUCUyG+8YQFUpuQyfGFxceaNCv0JzQ1mSCWMOBlV3kVapM7/7/b6SDGuXtMg/tLpyMyC4HAXgXGWCE0xqoXfnw1DSLnFihVFN/PxKN0bkGL72YZha19jO/0eZvPF//46UmfiJTFG7Bk98QtyQcsf3Ba233l48J7VQBtBKDWr2faOJ6VTfEP2jLrC7/f9I41N8Fg9Vl96dbd0NW6qQmAm+DCJQPwJsA8ryKlrnpRMjGkcGvmPEgArWf+8+Zn9cCAwEAAaNjMGEwDgYDVR0PAQH/BAQDAgIEMA8GA1UdEwEB/wQFMAMBAf8wHQYDVR0OBBYEFB3OJz+e4EpjwAyORQF7c+5fn7YIMB8GA1UdIwQYMBaAFB3OJz+e4EpjwAyORQF7c+5fn7YIMA0GCSqGSIb3DQEBCwUAA4IBAQCyf0RQN1QcfA34i0fFBAD2DHGGwOFjg+RDfIGIUCC3pPjr3nlcsrssIiYP+uwlV7FsjV4LJSl0o7fIx9r36fabrULoIIt6vfOAjOMbpuKZuh8Hm0RUEuuIBW+xH1Ymk4tSoFUPw10PnaR9i4svRRKJ/6WPgfcdHrf1yA39LkRMPH5075K7rkJHumClqviiFopvRCpZfoowZ+59bH10oEPZFCnJK8xq9g+o0P9uE3GgtJRNI6/3XXk9K7SzAEfm5uOsBFfmzkqQa/otrfiGvM2FthCttZY0W+EF1PLKstQ6lXrT7+wQDptuyaXfjxL2COO725qDAVc+nSaGrp4FAISX"],"kid":"ar2Me88UijW9Ha+DlQ1h9ciRs11y4x8k9wBaBPS0g/A=","kty":"RSA"},{"x5c":["MIIF5jCCA86gAwIBAgITMwAAAAPtVjxBz8HHIwAAAAAAAzANBgkqhkiG9w0BAQsFADCBgzELMAkGA1UEBhMCVVMxEzARBgNVBAgTCldhc2hpbmd0b24xEDAOBgNVBAcTB1JlZG1vbmQxHjAcBgNVBAoTFU1pY3Jvc29mdCBDb3Jwb3JhdGlvbjEtMCsGA1UEAxMkTWljcm9zb2Z0IEF6dXJlIEF0dGVzdGF0aW9uIFBDQSAyMDE5MB4XDTE5MDYwNTE3MjkwNVoXDTIwMDkwNTE3MjkwNVowfzELMAkGA1UEBhMCVVMxEzARBgNVBAgTCldhc2hpbmd0b24xEDAOBgNVBAcTB1JlZG1vbmQxHjAcBgNVBAoTFU1pY3Jvc29mdCBDb3Jwb3JhdGlvbjEpMCcGA1UEAxMgTWljcm9zb2Z0IEF6dXJlIEF0dGVzdGF0aW9uIDIwMTkwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCa+24LKNyEoAULNoU3nmVw2/4Xs7NUK+70v85lJrMDceHodyaBWQLfVXhz7/PD1bUfzT9bg+uxTNJ9NhTQnDf8hpTiuiMB60nS2PvzuN29tXdHmmROz+Ccu7yn7NyWB1ETOPhWP9I8tzk3K7BuJBAA7qyMBKasllDyaaW8haNIW6qyLPQvoBuK1I3idmXPwxOnwlESoxnylU1asa5cdlPb/CkaD8gPJuih0FN9k5C6Shnk2ijsmIrUJEuSkp/lZ1pBK91V9AWpsFLUctxCTM8tMZnPx4jygs7xZEpr/HLQBExOnCLam3/7BEW0fbB+WbjQflUv14ZYEAJM8U0FUK03AgMBAAGjggFUMIIBUDAOBgNVHQ8BAf8EBAMCB4AwFQYDVR0lBA4wDAYKKwYBBAGCN0wyAzAMBgNVHRMBAf8EAjAAMB0GA1UdDgQWBBRbD0G9SBlcr5cNUnSZMdC7KCpAMjAfBgNVHSMEGDAWgBStR15sz6nVWnU1XfoooXV4KJ9xrTBlBgNVHR8EXjBcMFqgWKBWhlRodHRwOi8vd3d3Lm1pY3Jvc29mdC5jb20vcGtpb3BzL2NybC9NaWNyb3NvZnQlMjBBenVyZSUyMEF0dGVzdGF0aW9uJTIwUENBJTIwMjAxOS5jcmwwcgYIKwYBBQUHAQEEZjBkMGIGCCsGAQUFBzAChlZodHRwOi8vd3d3Lm1pY3Jvc29mdC5jb20vcGtpb3BzL2NlcnRzL01pY3Jvc29mdCUyMEF6dXJlJTIwQXR0ZXN0YXRpb24lMjBQQ0ElMjAyMDE5LmNydDANBgkqhkiG9w0BAQsFAAOCAgEAZXRiVZbZl/Wiw1n1J4HDz2zRHSCLbzrwUc04Xh4QRYjDKgdA9dcALpcNQjM1If+wXaXQzbmDsW+5SJDg/9IcagKTPrUK/2pQ+/+uu2b4FsEMbNdq4thUnbIv+JXCFcLB2xjfVaSLhOnwtsNHO/QdzF739jsNuJ/YVz1OCIhxmK8pZWr/MjH8Q7Z2/1VHv6D5Sz/QVX9TIPJIMmvH1RVeoVlMXnGFsw0rUBD41lP6HR/lNNDGWo2OFo6ogsKNmlrO4s+vs6WX3eJDgT9K4cdHrZsJyKiCbaxR2e4n/7L+umMYewX/h392pzKOoEo36+6o8uDy6s2Uv2pn5xzx1PhgbR6w4+xkUFyfAuy6CRpl604R0aec2VtRwJFcJUDhfUaxIWWHYE5hMngVdXDgIcqG/21+/wtnqd/nIZyx0tgKo4gdnhQ63qnq9wuG39XjwmY7OFcd/8cMziha/BWnyYXEtFtzSEL5MfFpGNJAHk8hHJVDQUTXL6Cji1f4ha4QvcNS1pnMu2TV5t9dIx/j2d1BJjoMVB+cLxTonfbshq0EgSi9H1A3S6j0AYcbFCkWyeyZEqXFkEfFxpC9CCTY1y3nWSpgVbQ2LyDuEMnC5eEMoqPj0fTCHaYNX2EBoucgwtdkgEvyvoCyGKSrtpNY6Np5XrYF5+eSz+njF2Ym/KY/Z64=","MIIHQDCCBSigAwIBAgITMwAAADd1bHkqKXnfPQAAAAAANzANBgkqhkiG9w0BAQsFADCBiDELMAkGA1UEBhMCVVMxEzARBgNVBAgTCldhc2hpbmd0b24xEDAOBgNVBAcTB1JlZG1vbmQxHjAcBgNVBAoTFU1pY3Jvc29mdCBDb3Jwb3JhdGlvbjEyMDAGA1UEAxMpTWljcm9zb2Z0IFJvb3QgQ2VydGlmaWNhdGUgQXV0aG9yaXR5IDIwMTEwHhcNMTkwNTMwMjI0ODUyWhcNMzQwNTMwMjI1ODUyWjCBgzELMAkGA1UEBhMCVVMxEzARBgNVBAgTCldhc2hpbmd0b24xEDAOBgNVBAcTB1JlZG1vbmQxHjAcBgNVBAoTFU1pY3Jvc29mdCBDb3Jwb3JhdGlvbjEtMCsGA1UEAxMkTWljcm9zb2Z0IEF6dXJlIEF0dGVzdGF0aW9uIFBDQSAyMDE5MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAyTLy/bGuzAnrxE+uLoOMwDbwVj/TlPUSeALDYWh1IEV1XASInpSRVgacIHDFfnIclB72l7nzZuRjrsmnNgG0H/uDj0bs+AZkxZ6si/E0E3KOP8YEYSOnDEuCfrBQDdye62tXtP3WAhFe88dW6p56pyxrG1BgpnIsDiEag4U6wzmjkWrFM2w5AFbYUiyloLrr6gnG2Cuk4pTkLW6k3qXo/Nfjm+bS/wgtfztM3vi3lsM4nJvB0HEk8coUQxobpmigmQxBRz7OZH99oWYn9XDR1bym0G/nJ/+Y95Z6YquguLk4YHQ8QrXpAf8/dyRQe3zeQu387CLCksmxYTVaGE3QCQEx2M3dIUmUiFiJSgGO7wsq+tf3oqT39GXP6ftdhE6V1UcX/YgK4SjIcxuD7Sj9RW+zYq3iaCPIiwjSK+MFwLtLdMZUmzmXKPmz2sW5rj4Jh6jcmLVc+a6xccE3x0nQXTTCFNlQRCMqP7GYSaMzjfq2m4leCqunaLG3m6XPOxlKQqAsFvNWxWw0ujV8ILUpo9ZattvHrIukv5/IvK4YCrbeyQUEi1aQzokGGGnKwDWNwCwoEwtVV3CJ7Mw6Gvqk6JuxbixGIE/vSjwnSaal8OdBCQqZHTHSbkaVYJlVaVDjZQtj01RmCQjJmJlzYGTrsMwK9y/DMd8tVyxfYVPc+G8CAwEAAaOCAaQwggGgMA4GA1UdDwEB/wQEAwIBhjAQBgkrBgEEAYI3FQEEAwIBADAdBgNVHQ4EFgQUrUdebM+p1Vp1NV36KKF1eCifca0wVAYDVR0gBE0wSzBJBgRVHSAAMEEwPwYIKwYBBQUHAgEWM2h0dHA6Ly93d3cubWljcm9zb2Z0LmNvbS9wa2lvcHMvRG9jcy9SZXBvc2l0b3J5Lmh0bTAZBgkrBgEEAYI3FAIEDB4KAFMAdQBiAEMAQTAPBgNVHRMBAf8EBTADAQH/MB8GA1UdIwQYMBaAFHItOgIxkEO5FAVO4eqnxzHRI4k0MFoGA1UdHwRTMFEwT6BNoEuGSWh0dHA6Ly9jcmwubWljcm9zb2Z0LmNvbS9wa2kvY3JsL3Byb2R1Y3RzL01pY1Jvb0NlckF1dDIwMTFfMjAxMV8wM18yMi5jcmwwXgYIKwYBBQUHAQEEUjBQME4GCCsGAQUFBzAChkJodHRwOi8vd3d3Lm1pY3Jvc29mdC5jb20vcGtpL2NlcnRzL01pY1Jvb0NlckF1dDIwMTFfMjAxMV8wM18yMi5jcnQwDQYJKoZIhvcNAQELBQADggIBABNiL5D1GiUih16Qi5LYJhieTbizpHxRSXlfaw/T0W+ow8VrlY6og+TT2+9qiaz7o+un7rgutRw63gnUMCKtsfGAFZV46j3Gylbk2NrHF0ssArrQPAXvW7RBKjda0MNojAYRBcrTaFEJQcqIUa3G7L96+6pZTnVSVN1wSv4SVcCXDPM+0D5VUPkJhA51OwqSRoW60SRKaQ0hkQyFSK6oGkt+gqtQESmIEnnT3hGMViXI7eyhyq4VdnIrgIGDR3ZLcVeRqQgojK5f945UQ0laTmG83qhaMozrLIYKc9KZvHuEaG6eMZSIS9zutS7TMKLbY3yR1GtNENSTzvMtG8IHKN7vOQDad3ZiZGEuuJN8X4yAbBz591ZxzUtkFfatP1dXnpk2YMflq+KVKE0V9SAiwE9hSpkann8UDOtcPl6SSQIZHowdXbEwdnWbED0zxK63TYPHVEGQ8rOfWRzbGrc6YV1HCfmP4IynoBoJntQrUiopTe6RAE9CacLdUyVnOwDUJv25vFU9geynWxCRT7+yu8sxFde8dAmB/syhcnJDgQ03qmMAO3Q/ydoKOX4glO1ke2rumk6FSE3NRNxrZCJ/yRyczdftxp9OP16M9evFwMBumzpy5a+d3I5bz+kQKqsr7VyyDEslVjzxrJPXVoHJg/BWCs5nkfJqnISyjC5cbRJO","MIIF7TCCA9WgAwIBAgIQP4vItfyfspZDtWnWbELhRDANBgkqhkiG9w0BAQsFADCBiDELMAkGA1UEBhMCVVMxEzARBgNVBAgTCldhc2hpbmd0b24xEDAOBgNVBAcTB1JlZG1vbmQxHjAcBgNVBAoTFU1pY3Jvc29mdCBDb3Jwb3JhdGlvbjEyMDAGA1UEAxMpTWljcm9zb2Z0IFJvb3QgQ2VydGlmaWNhdGUgQXV0aG9yaXR5IDIwMTEwHhcNMTEwMzIyMjIwNTI4WhcNMzYwMzIyMjIxMzA0WjCBiDELMAkGA1UEBhMCVVMxEzARBgNVBAgTCldhc2hpbmd0b24xEDAOBgNVBAcTB1JlZG1vbmQxHjAcBgNVBAoTFU1pY3Jvc29mdCBDb3Jwb3JhdGlvbjEyMDAGA1UEAxMpTWljcm9zb2Z0IFJvb3QgQ2VydGlmaWNhdGUgQXV0aG9yaXR5IDIwMTEwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQCygEGqNThNE3IyaCJNuLLx/9VSvGzH9dJKjDbu0cJcfoyKrq8TKG/Ac+M6ztAlqFo6be+ouFmrEyNozQwph9FvgFyPRH9dkAFSWKxRxV8qh9zc2AodwQO5e7BW6KPeZGHCnvjzfLnsDbVU/ky2ZU+I8JxImQxCCwl8MVkXeQZ4KI2JOkwDJb5xalwL54RgpJki49KvhKSn+9GY7Qyp3pSJ4Q6g3MDOmT3qCFK7VnnkH4S6Hri0xElcTzFLh93dBWcmmYDgcRGjuKVB4qRTufcyKYMME782XgSzS0NHL2vikR7TmE/dQgfI6B0S/Jmpaz6SfsjWaTr8ZL22CZ3K/QwLopt3YEsDlKQwaRLWQi3BQUzK3Kr9j1uDRprZ/LHR47PJf0h6zSTwQY9cdNCssBAgBkm3xy0hyFfj0IbzA2j70M5xwYmZSmQBbP3sMJHPQTySx+W6hh1hhMdfgzlirrSSL0fzC/hV66AfWdC7dJse0Hbm8ukG1xDo+mTeacY1logC8Ea4PyeZb8txiSk190gWAjWP1Xl8TQLPX+uKg09FcYj5qQ1OcunCnAfPSRtOBA5jUYxe2ADBVSy2xuDCZU7JNDn1nLPEfuhhbhNfFcRf2X7tHc7uROzLLoax7Dj2cO2rXBPB2Q8Nx4CyVe0096yb5MPa50c8prWPMd/FS6/r8QIDAQABo1EwTzALBgNVHQ8EBAMCAYYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUci06AjGQQ7kUBU7h6qfHMdEjiTQwEAYJKwYBBAGCNxUBBAMCAQAwDQYJKoZIhvcNAQELBQADggIBAH9yzw+3xRXbm8BJyiZb/p4T5tPw0tuXX/JLP02zrhmu7deXoKzvqTqjwkGw5biRnhOBJAPmCf0/V0A5ISRW0RAvS0CpNoZLtFNXmvvxfomPEf4YbFGq6O0JlbXlccmh6Yd1phV/yX43VF50k8XDZ8wNT2uoFwxtCJJ+i92Bqi1wIcM9BhS7vyRep4TXPw8hIr1LAAbblxzYXtTFC1yHblCk6MM4pPvLLMWSZpuFXst6bJN8gClYW1e1QGm6CHmmZGIVnYeWRbVmIyADixxzoNOieTPgUFmG2y/lAiXqcyqfABTINseSO+lOAOzYVgm5M0kS0lQLAausR7aRKX1MtHWAUgHoyoL2n8ysnI8X6i8msKtyrAv+nlEex0NVZ09Rs1fWtuzuUrc66U7h14GIvE+OdbtLqPA1qibUZ2dJsnBMO5PcHd94kIZysjik0dySTclY6ysSXNQ7roxrsIPlAT/4CTL2kzU0Iq/dNw13CYArzUgA8YyZGUcFAenRv9FO0OYoQzeZpApKCNmacXPSqs0xE2N2oTdvkjgefRI8ZjLny23h/FKJ3crWZgWalmG+oijHHKOnNlA8OqTfSm7mhzvO6/DggTedEzxSjr25HTTGHdUKaj2YKXCMiSrRq4IQSB/c9O+lxbtVGjhjhE63bK2VVOxlIhBJF7jAHscPrFRH"],"kid":"ZQzLfoNwhBNCu9r8Fucjn6H4Cdw","kty":"RSA"}]}`

Can you share the URL where you got that response from?

Reply:
just update the example json